src/libpriv/rpmostree-core.c

```diff
@@ -444,6 +423,14 @@ rpmostree_context_new_unprivileged (int basedir_dfd,
   return rpmostree_context_new_internal (basedir_dfd, TRUE, cancellable, error);
 }
 
+/* Use this if no packages will be installed, and we just want a "dummy" run.
+ */
+void
+rpmostree_context_set_is_empty (RpmOstreeContext *self)
```

Comment:
Yeah - almost.  Look at the one caller of it.  Basically libdnf aborts if there's no sack.